build/linux/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -164,6 +204,16 @@ def populateSettingValuesFromConfigMap(parsedConfig)
   file.write("export HPA_CHUNK_SIZE=#{@hpaChunkSize}\n")
   file.write("export PODS_EMIT_STREAM_BATCH_SIZE=#{@podsEmitStreamBatchSize}\n")
   file.write("export NODES_EMIT_STREAM_BATCH_SIZE=#{@nodesEmitStreamBatchSize}\n")
+  # fbit settings
+  if @fbitFlushInterval > 0
+    file.write("export FBIT_SERVICE_FLUSH_INTERVAL=#{@fbitFlushInterval}\n")
+  end
+  if @fbitTailBufferChunkSize > 0
+    file.write("export FBIT_TAIL_BUFFER_CHUNK_SIZE=#{@fbitTailBufferChunkSize}\n")
+  end
+  if @fbitTailBufferMaxSize > 0
+    file.write("export FBIT_TAIL_BUFFER_MAX_SIZE=#{@fbitTailBufferMaxSize}\n")
```

Comment:
As we didnt have a dependency between any parser scripts before, we need to add a note in the main.sh file so that we don't move around the scripts and miss the ordering.

Reply:
> As we didnt have a dependency between any parser scripts before, we need to add a note in the main.sh file so that we don't move around the scripts and miss the ordering.

> great catch. completely make sense. let me add the comment.

Reply:
Fixed.